lib/NAnt/NAnt.exe.config

```diff
@@ -721,7 +721,7 @@
                         <include name="Microsoft.CSharp.dll" />
                         <include name="Microsoft.JScript.dll" />
                         <include name="Microsoft.VisualBasic.Compatibility.Data.dll" />
-                        <include name="Microsoft.VisualBasic.Comptatibility.dll" />
+                        <include name="Microsoft.VisualBasic.Compatibility.dll" />
```

Comment:
This is very strange. 

Google doesn't think many people know about this.
https://www.google.com/search?q=%22Microsoft.VisualBasic.Comptatibility.dll%22&oq=%22Microsoft.VisualBasic.Comptatibility.dll%22&aqs=chrome.0.69i59j0i13i30l5j0i8i13i30.657j0j9&sourceid=chrome&ie=UTF-8